third_party/terraform/resources/resource_google_billing_subaccount.go

```diff
@@ -0,0 +1,132 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"google.golang.org/api/cloudbilling/v1"
+)
+
+func resourceBillingSubaccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceBillingSubaccountCreate,
+		Read:   resourceBillingSubaccountRead,
+		Delete: resourceBillingSubaccountDelete,
+		Update: resourceBillingSubaccountUpdate,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"display_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"open": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"master_billing_account": {
+				Type:             schema.TypeString,
+				Required:         true,
+				ForceNew:         true,
+				DiffSuppressFunc: compareSelfLinkOrResourceName,
+			},
+			"billing_account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"rename_on_destroy": {
```

Comment:
Instead of `{{action}}_on_destroy`, can you call this field `deletion_policy` and allow `""` and `"RENAME_ON_DESTROY"` to be specified? https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_user#deletion_policy is a good model, where we've done the same.

Reply:
I looked at that resource and I think I have implemented it as you have requested